lib/stdlib/doc/src/gen_server.xml

```diff
@@ -546,7 +554,8 @@ gen_server:abcast     -----> Module:handle_cast/2
           the function returns <c>{error,Reason}</c>. If
           <c>Module:init/1</c> returns <c>{stop,Reason}</c> or
           <c>ignore</c>, the process is terminated and the function
-          returns <c>{error,Reason}</c> or <c>ignore</c>, respectively.</p>
+          returns <c>{error,Reason}</c> or <c>ignore</c>, respectively.
+	  An exit signal with the same reason is emitted.</p>
```

Comment:
If `ignore` is returned the `gen_server` process exits with exit reason `normal`. I think it also should mention here that exit signals are sent to all linked processes and ports.

Reply:
Changing to "`An exit signal with the same <c>Reason</c> (or <c>normal</c> if
	  <c>Module:init/1</c> returns <c>ignore</c>) is sent to linked
	  processes and ports.`"